packages/teams-js/prepBetaRelease.js

```diff
@@ -75,10 +75,12 @@ function getNewerPrefix(currBetaVer, currPkgJsonVer) {
   const pkgParts = currPkgPrefix.split('.');
 
   for (let i = 0; i < betaParts.length; i++) {
-    if (betaParts[i] > pkgParts[i]) {
+    const betaPart = Number(betaParts[i]);
+    const pkgPart = Number(pkgParts[i]);
+    if (betaPart > pkgPart) {
       return currBetaPrefix;
     }
-    if (pkgParts[i] > betaParts[i]) {
+    if (pkgPart > betaPart) {
       return currPkgPrefix;
     }
   }
```

Comment:
I know that this isn't something you're changing as part of this but would it be prudent to put code after this for loop that throws/crashes/whatever to indicate that we got to a place in the function that we never expected to get to?

Reply:
Good idea, since this was merged I will put out a separate PR with those changes.